packaging/installer/README.md

```diff
@@ -234,6 +234,15 @@ package|description
 
 *Netdata will greatly benefit if you have the above packages installed, but it will still work without them.*
 
+package|description
+:-----:|-----------
+`libuv`|multi-platform support library with a focus on asynchronous I/O
+`liblz4`|Extremely Fast Compression algorithm
```

Comment:
The package has a different name in centos, probably needs to be clear here how to find it per distro

Reply:
These names are the official package names from the developers. The more distro specific names are in configure.ac in the error messages.